### ISSUE TITLE
3.0.0-beta.22 Sortable placeholder class not removed on drop

### ISSUE THREAD

**Issue by mktcode** [uikit/uikit#2658]
3.0.0-beta.22
Firefox 53, Linux (Mint 18)

http://codepen.io/anon/pen/LyQGrr

Click on "Structures" and drag an item to the main area. The sortable is configured to show a placeholder from the placeholder component while dragging.

`<div uk-sortable="group: sortable-group; cls-drag: uk-card-primary uk-sortable-drag; cls-placeholder: uk-placeholder">`

On dropping the item I expected the placeholder class to be removed again but it stays on the element while cls-drag class is removed as expected.

This is my workaround for now:

```
$(function() {
    $("#website").on("change", function(event, options, item) {
      $(item).removeClass("uk-placeholder");
    });
});
```

I wonder if this really is a bug or if I am missing something... ???

**Comment by fabien** [comment]
You should add `cls-placeholder: uk-placeholder` to `#website` as well, in order for this sortable to know which class to remove. 

Perhaps it can be improved within Uikit to use the same settings within groups automatically.

**Comment by mktcode** [comment]
Ok, works! Thank you!

**Comment by janschoenherr** [comment]
Thanks. Resolved with 25a6752fd85606403cfb148b9651b857dff8c8ed
